S/SCS_MKL/build_tarballs.jl

```diff
@@ -37,7 +38,7 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("MKL_jll"; compat="2022.2.0"),
+    BuildDependency(PackageSpec(;name="MKL_jll", version=v"2022.2")),
```

Comment:
@kalmarek latest version is 2023.1. Do we want to update to that? x-ref https://github.com/jump-dev/SCS.jl/issues/274

Reply:
We do ;) But first we'd need to come to a conclusion what to do with the `BuildDependency` vs `Dependency`.

Updating the version of MKL_jll is a trivial thing ;)

Reply:
```suggestion
    BuildDependency(PackageSpec(;name="MKL_jll", version=v"2023.1")),
```